Multisig/UI/Settings/OwnerKeyManagement/KeyAddedViewController.swift

```diff
@@ -66,6 +66,7 @@ fileprivate extension KeyType {
         case .deviceGenerated: return "Generate Owner Key"
         case .keystone: return "Connect Keystone"
         case .walletConnect: return "Connect WalletConnect"
+        case .web3Auth: return "Login via Web2"
```

Comment:
Is that the wording from the design? Do we even have a design for this?

Reply:
No, we don't have design yet, I just add it for now and later we change once we have the design, however this case won't be triggered for now